css/components/spacing.css

```diff
@@ -0,0 +1,134 @@
+.mg-bottom-xs {
+  margin-bottom: 1.6rem;
+}
+
+.mg-bottom-s {
+  margin-bottom: 2.4rem;
+}
+
+.mg-bottom-m {
+  margin-bottom: 4.8rem;
+}
+
+.mg-bottom-l {
+  margin-bottom: 8rem;
+}
+
+.mg-bottom-xl {
+  margin-bottom: 12.8rem;
+}
+
+/******** Margin-Top ********/
+
+.mg-top-s {
+  margin-top: 2.4rem;
+}
+
+.mg-top-s-m {
+  margin-top: 3.2rem;
+}
+
+.mg-top-s-l {
+  margin-top: 4.8rem;
+}
+
+.mg-top-s-xl {
+  margin-top: 6.4rem;
+}
+
+/******** Margin-Left ********/
+
+.mg-left-s {
+  margin-left: 2.4rem;
+}
+
+.mg-left-m {
+  margin-left: 3.2rem;
+}
+
+.mg-left-l {
+  margin-left: 4.8rem;
+}
+
+.mg-left-xl {
+  margin-left: 6.4rem;
+}
+
+/******** Margin-Right ********/
+
+.mg-right-s {
+  margin-right: 2.4rem;
+}
+
+.mg-right-m {
+  margin-right: 3.2rem;
+}
+
+.mg-right-l {
+  margin-right: 4.8rem;
+}
+
+.mg-right-xl {
+  margin-right: 6.4rem;
+}
+
+/******** Padding-Bottom ********/
+
+.pd-bottom-s {
```

Comment:
Do not leave these classes as empty!!Naming of these classes is really great!!

Reply:
woops! Thank you for catching that - I will fix it right away